code/hsec-sync/src/Security/Advisories/Sync/Url.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE LambdaCase #-}
```

Comment:
if you activate these in every module perhaps consider moving them to the `.cabal` file